templates/docker-compose.yaml.j2

```diff
@@ -35,6 +41,8 @@ services:
       - fluentbit
     labels:
       - "container=hoprd"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
Please remove, we use the static IP to workaround the limitation of this extra host setting

Reply:
done